docs/reference/6-data-import/technical-articles/bulk-delete-using-data-import.md

```diff
@@ -0,0 +1,39 @@
+# Bulk Delete using Data Import
+
+Beginning with [v2.2](https://edfi.atlassian.net/wiki/pages/viewpage.action?pageId=24119656#What%27sNewinDataImport(2.2precopy)-What%27sNewinDataImportv2.1) the Data Import tool has the capability to bulk delete resources from csv files. This functionality is supported in two ways, **delete by Id** and **delete by natural key**. If you have access to the primary key `Id` of the resources you wish to delete, deleting by Id is simpler and faster. However, if you do not have access to that data, you can easily map to the natural key members of the resource you wish to delete.
```

Comment:
Please replace with a local link. In general, just search for "atlassian.net" within the `docs/reference/6-data-import` folder to find everything that needs to be modified.